src/templates/landing-template.js

```diff
@@ -0,0 +1,48 @@
+// @flow strict
+import React from 'react';
+import Layout from '../components/Layout';
+import Page from '../components/Page';
+import Landing from '../components/Landing/Landing';
+import type { Edges } from '../types';
+
+type Props = {
+  pageContext: {
+    lastPosts: {
+      data: {
+        allMarkdownRemark: {
+          edges: Edges
+        }
+      }
+    },
+    clients: {
+      data: {
+        allFile: {
+          edges: Edges
+        }
+      }
+    },
+    skills: {
+      data: {
+        allFile: {
+          edges: Edges
+        }
+      }
+    }
+  }
+}
+
+const LandingTemplate = ({ pageContext }: Props) => {
+  const lastPosts = pageContext.lastPosts.data.allMarkdownRemark.edges;
+  const clients = pageContext.clients.data.allFile.edges;
+  const skills = pageContext.skills.data.allFile.edges;
```

Comment:
What is these values are `null` or `undefined`. Here is the place to check that everything worked as expected before passing the data to other Components.
ALWAYS check for limit cases (what if something is null or infinite).